src/Common/parameters.jl

```diff
@@ -1,21 +1,45 @@
-"""
-    Create common model parameters.
-
-    (Overwriting the default values happens in the driver file.
-    Here we are just creating the CommonParameters struct)
-"""
 module Parameters
 
+using DocStringExtensions
+import ClimaParams as CP
+
 abstract type AbstractCommonParameters end
 const ACP = AbstractCommonParameters
 
-# Define KiD parameters
+"""
+    CommonParameters{FT}
+
+    Common parameters for the kinematic driver simulations
+
+    #Fields
+    $(DocStringExtensions.FIELDS)
+"""
 Base.@kwdef struct CommonParameters{FT} <: ACP
-    precip_sources::Int
-    precip_sinks::Int
+    "Switch to include precipitation formation terms"
+    precip_sources::Bool
+    "Switch to include precipitation sink terms"
+    precip_sinks::Bool
+    "Prescribed number concentration of cloud droplets (needed for some schemes) [1/m3]"
     prescribed_Nd::FT
 end
 
+function CommonParameters(td::CP.AbstractTOMLDict)
+    #name_map = (;
```

Comment:
@nefrathenrici - could you take a look at why the map didn't work here?

Reply:
Should be same as above, the parameters need to be splatted